apps/fyler/src/fyler_event_listener.erl

```diff
@@ -54,8 +39,10 @@ handle_event(#fevent{type = failed, node = Node, task = #task{id=TaskId, file =
         true ->
           ets:delete(Category, TaskId),
           fyler_server:save_task_stats(Stats),
-          fyler_server:send_response(Task,undefined,failed),
-          gen_server:cast(fyler_server, {task_finished,Node}),
+          fyler_server:send_response(Task, Stats),
+          gen_server:cast(fyler_server, {task_finished, Node}),
+          ToLogstash = [TaskId, Result, Category, Type, Url, Size, Node, DTime, Time, UTime],
```

Comment:
Line 44 is too long:           ToLogstash = [TaskId, Result, Category, Type, Url, Size, Node, DTime, Time, UTime],.